docs/book/src/reference/versions.md

```diff
@@ -15,19 +15,26 @@ All Infrastructure Providers are maintained by independent teams. Other Bootstra
 
 ## Supported Kubernetes Versions
 
-The project aims to keep the current minor release compatible with the actively supported Kubernetes minor releases, i.e.,
-the current release (N), N-1, and N-2. Where possible we will go above and beyond this trying to help our users support
-both newer and older versions of Kubernetes:
-
-- For a given Cluster API minor release, we give best-effort support for future Kubernetes versions that are released
-	during the release's active support period. Depending on the changes made in Kubernetes, users may be required to
-	update to the next patch release of Cluster API to maintain compatibility.
-- In some cases where support for future Kubernetes versions can't be added, or is considered not practical or too risky
-	adding it, the Kubernetes version support matrix for an existing Cluster API minor release will stop at the last supported version.
-- Cluster API offers best-effort support for older Kubernetes versions where possible.
-	Important! this is not a replacement/alternative for upstream Kubernetes support policies!
-	Support for older version of Kubernetes is limited to "Cluster API can start a Cluster with this Kubernetes version"
-	and "Cluster API  can upgrade to the next Kubernetes version"; it does not include any extended support to Kubernetes itself.
+A Cluster API minor release supports:
+* 4 Kubernetes minor releases for the management cluster (N - N-3)
+* 6 Kubernetes minor releases for the workload cluster (N - N-5)
+
+When a new Kubernetes minor release is available, it will be supported in an upcoming Cluster API patch release.
```

Comment:
I think we need to keep a disclaimer about the fact that adding support for future releases depends on the changes made in Kubernetes, and if those changes are too invasive, support for those Kubernetes releases will be deferred to the next minor.
(there was a note about this in the previous version)

Reply:
Sounds fine

I dropped it mostly because I didn't want to keep the part that we backport the support into every branch, but that disclaimer is fine